examples/workload/overlay/main.tf

```diff
@@ -112,12 +112,17 @@ locals {
   trunk_2_subnet_id  = aws_subnet.data[2].id
   access_b_subnet_id = aws_subnet.data[3].id
 
-  default_image_repository = format(
-    "%s.dkr.ecr.%s.amazonaws.com/xrd/xrd-vrouter",
+  default_image_registry = format(
```

Comment:
Is something prefixed with "default_" automatically takes as the value for a variable if not overridden?

Reply:
Oh ignore me, I guess this is done explicitly with `coalesce`